rpcs3/Emu/Cell/PPUTranslator.cpp

```diff
@@ -3833,7 +3833,21 @@ void PPUTranslator::FMADDS(ppu_opcode_t op)
 	const auto a = GetFpr(op.fra);
 	const auto b = GetFpr(op.frb);
 	const auto c = GetFpr(op.frc);
-	const auto result = m_ir->CreateFPTrunc(m_ir->CreateFAdd(m_ir->CreateFMul(a, c), b), GetType<f32>());
+
+	llvm::Value* result;
+	if(m_use_fma)
```

Comment:
```suggestion
	if (m_use_fma)
```
ditto for the rest of the `if`s in this PR

Reply:
Should be done now, I think.